.automation/generated/linter-helps.json

```diff
@@ -8155,7 +8155,7 @@
         "  scanner run dfa  scan codebase with all DFA rules",
         "",
         "WARNING: We're continually improving Salesforce Code Analyzer. Tell us what you think! Give feedback at https://research.net/r/SalesforceCA",
-        "(node:1830) Warning: Deprecated config name: apiVersion. Please use org-api-version instead.",
+        "(node:1822) Warning: Deprecated config name: apiVersion. Please use org-api-version instead.",
```

Comment:
@nvuillam our salesforce guy, why would they have this message when calling for help, is it that important?

Reply:
SF plugins are not that shiny... I wouldn't bother about this one ;)